gnosis/safe-react
# [Address Book v2] Fix AB v2 migration

## What it solves
Resolves #2315.

There was a bug that we didn't migrate the safe owners. It used to be an object with name and address, and now it's just an address string.

Because of this bug, ImmortalDB was writing an empty `{}` into the SAFES storage.

## How this PR fixes it
It now maps owners to addresses.

## How to test it
Create (or update) a localStorage key `_immortal|v2_RINKEBY__SAFES` and paste the content of this key on production.

Same with the `_immortal|v2_RINKEBY__ADDRESS_BOOK_STORAGE_KEY` key.

After a page reload, `_immortal|v2_RINKEBY__SAFES` should be migrated, and `_immortal|v2_RINKEBY__ADDRESS_BOOK_STORAGE_KEY` completely removed.
